challenge_7/python/aemc/src/find_missing_number.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+
+
+def find_missing_number(nums):
+    # sort list first and assign it to new variable called newlist
+    newlist = sorted(nums)
+    return sum(xrange(len(newlist)+1)) - sum(newlist) 
```

Comment:
One of the stipulations of this challenge is that you _must not_ sort the given list when determining the solution. This may give you the correct result of finding the missing number, but it bypasses the rules of the challenge.